### ISSUE TITLE
Pushing RTMP-stream to Twitch-channel with Disconnect-protection enabled

### ISSUE THREAD

**Issue by DigibitDK** [arut/nginx-rtmp-module#1595]
I generel, I have settings that uses 'push rtmp://<twitch-server>/app/<key> and it works fine.
But if the Twitch-channel has Disconnect-protection enabled, this will kick in, when I stop the incoming stream to NGINX.
So, Twitch thinks that the stream has been interrupted, rather then just being stopped.
Of course, I can just disable this behavior in Twitch, but otherwise, the only way I can come across this, is to re-encode it, and make ffmpeg push it to Twitch with libx264. Then Twitch knows, that stream is closed, and not just interrupted.

Is it really not possible to make this work correctly, just by using NGINX and PUSH?

/Michael.

**Comment by duncte123** [comment]
I tried a lot of different options to make sure all headers get forwarded etc and it seems that the module does not forward the end signal to all `push` services. This should be fixed as this is a major flaw imo.

**Comment by 0lhi** [comment]
>This should be fixed as this is a major flaw imo.

Agreed. Getting into Multistreaming currently, and this is a potential dealbreaker.

**Comment by duncte123** [comment]
This could be a one line fix. @arut please have a look at it.

**Comment by Takabrycheri** [comment]
Any update on this?

**Comment by fuyonet** [comment]
any news here ?
